Assets/AppCenter/Editor/AppCenterSettingsEditorAdvanced.cs

```diff
@@ -13,7 +13,7 @@ public override void OnInspectorGUI()
         serializedObject.Update();
 
         EditorGUILayout.PropertyField(serializedObject.FindProperty("TransmissionTargetToken"));
-        EditorGUILayout.PropertyField(serializedObject.FindProperty("AppCenterStartupType"));
+        //EditorGUILayout.PropertyField(serializedObject.FindProperty("AppCenterStartupType"));
```

Comment:
@annakocheshkova Can you please add a comment why we need to keep this commented and do not delete this at all?